scripts/get-deploy-info.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$(cast chain --rpc-url="$ETH_RPC_URL")" == "ethlive" ]] || { echo "Please set a mainnet ETH_RPC_URL"; exit 1; }
+
+[[ "$1" =~ ^0x[[:xdigit:]]{66}$ ]] || { echo "Please specify the transaction to inspect (e.g. tx=0x<txhash>)"; exit 1; }
```

Comment:
This is not exactly as the goerli version, this seems more accurate though.

Reply:
yes, this check improvement was meant to be back-ported to `spells-goerli`